### PR TITLE
feat:Authetication 

### PR DESCRIPTION
**Pull request by shubhamkumar648**
1. add login and signup (Authentication)
2. refactor minor CSS

### CODE REVIEW

**Review by RahulSingh9131 — APPROVED** [review]
mentioned feature is working perfectly. PR is ready to merge👍

File: ecomerce-app/src/Reducers/SignupReducer.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (saketkum11) [inline ecomerce-app/src/Reducers/SignupReducer.js:1]: Without signup you can do sign form .

**Review by saketkum11 — APPROVED** [review]
- code is clean 
- much easy to understand
- better word choice

File: ecomerce-app/src/Reducers/SignupReducer.js (hunk 1 above)
Comment (shubhamkumar648) [reply ecomerce-app/src/Reducers/SignupReducer.js:1]: okay

**Pull request merged**